src/chapters/sql-part1/studio.rst

```diff
@@ -0,0 +1,269 @@
+Studio: Databases Part 1
+=========================
+
+A database fills the role of the model in an MVC structure.
+
+In this studio, you will use space in *MySQLWorkbench* to practice writing SQL
+queries to retrieve or modify information stored in an established database.
+You will also explore the parallels between Java objects and database tables.
+
+Walkthrough
+------------
+
+Let's relate what you've learned about SQL to interact with a database called
+``movie-buff`` that contains two tables---``movies`` and ``directors``.
+
+Just as you did in the :ref:`exercises <sql-part1-exercises>` create the new
+``movie-buff`` schema.
+
+Before we create the ``movies`` and ``directors`` tables, let's think about
+what kind of columns we want in each one.
+
+Movies Table
+^^^^^^^^^^^^
+
+For the ``movies`` table, it makes sense to have columns for the ``title`` of
+the movie, the ``year`` it was released, and the ``director``. We're also going
+to want to have a unique ``movie_id`` column as the primary key, since there
+can be more than one movie with the same title.
+
+Take the time now to think about what data types you might use for these
+columns (``INTEGER``, ``VARCHAR``, ``DATE``, etc.), then go ahead and write (or
+type) the SQL statement to create this table.
+
+As you consider which data types to use for your ``movies`` columns, you might
+find yourself thinking, *If this were a property of a Java object, what data
+type would I use?*
+
+For example, if the ``movie`` object has a ``title`` and a ``director`` field,
+you would expect both of these data types to be ``String``. Similarly the year
+the movie was released would be stored as an integer. This conceptual overlap
+between databases and Java objects is very useful, and we will explore this
+further in the class on *Object-Relational Mapping* (ORM).
+
+Here's how our ``movies`` table could be created:
+
+.. sourcecode:: sql
+   :linenos:
+
+   CREATE TABLE movies (
+      movie_id INTEGER PRIMARY KEY AUTO_INCREMENT,
```

Comment:
Maybe add a reference to the exercises? Or a note for the TAs or instructor to go over this line? I can see someone not doing the exercises and being confused.

Reply:
I added the link and an encouragement to discuss line 2 with a partner.